docker/entrypoint.sh

```diff
@@ -96,11 +96,20 @@ run_manage() {
 }
 
 
+redis_connection_hack() {
+    PULP_REDIS_URL="rediss://${PULP_REDIS_PASSWORD}@${PULP_REDIS_HOST:-localhost}:${PULP_REDIS_PORT:-6379}/0"
```

Comment:
1. Please select schema `redis` or `rediss` depending on a PULP_REDIS_SSL variable value.
2. The `PULP_REDIS_PASSWORD` parameter is optional. Please add in conditionally. Also it has to be preseded by semicolon.